testsuite/integration/basic/src/test/java/org/wildfly/test/integration/observability/micrometer/MicrometerSetupTask.java

```diff
@@ -0,0 +1,45 @@
+package org.wildfly.test.integration.observability.micrometer;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.IOException;
+
+import org.jboss.as.arquillian.api.ServerSetupTask;
+import org.jboss.as.arquillian.container.ManagementClient;
+import org.jboss.as.controller.client.helpers.Operations;
+import org.jboss.as.test.shared.ServerReload;
+import org.jboss.dmr.ModelNode;
+
+public class MicrometerSetupTask implements ServerSetupTask {
+    private final ModelNode extensionAddress = Operations.createAddress("extension", "org.wildfly.extension.micrometer");
+    private final ModelNode subsystemAddress = Operations.createAddress("subsystem", "micrometer");
+
+    @Override
+    public void setup(final ManagementClient managementClient, final String containerId) throws Exception {
+        execute(managementClient, Operations.createAddOperation(extensionAddress), true);
+        execute(managementClient, Operations.createAddOperation(subsystemAddress), true);
```

Comment:
Should not the metrics extension/subsystem be removed to perform this test? The documentation describes that metrics should be removed to use micrometer, so I'm pointing out here in case this test could fail because both extensions are added at the same time.

Reply:
The docs are actually a bit out of date. These two extensions can be run together, though I image there would be some performance impact. It wouldn't hurt, I guess, to remove it, as that is (or will be) the suggested configuration.